.meteor/versions

```diff
@@ -148,6 +148,7 @@ templating-tools@1.1.2
 tmeasday:check-npm-versions@0.3.2
 todda00:friendly-slugs@0.6.0
 tracker@1.2.0
+twbs:bootstrap@3.3.6
```

Comment:
Why do you need this?

Reply:
I have used glyphicon in search option it is updated package of bootstrap

Reply:
We have all the resources we need, natively.
Please, remove this dependency, thanks.

Reply:
Removed

Reply:
No, it's still there.